test/typescript/terraform-cloud/test.ts

```diff
@@ -6,14 +6,14 @@
  * @group asset
  */
 
-import { TestDriver } from "../../test-helper";
+import { TestDriver, onPosix } from "../../test-helper";
 import { TerraformCloud } from "@skorfmann/terraform-cloud";
 import * as crypto from "crypto";
 const { TERRAFORM_CLOUD_TOKEN, GITHUB_RUN_NUMBER, TERRAFORM_VERSION } =
   process.env;
-const withAuth = TERRAFORM_CLOUD_TOKEN ? it : it.skip;
+const withAuth = TERRAFORM_CLOUD_TOKEN ? onPosix : onPosix.skip;
```

Comment:
nit: Maybe call this `onPosixWithAuth` now? Or add a comment above the testcase that this test is currently disabled for windows?